ChangeLog.md

```diff
@@ -20,6 +20,9 @@ See docs/process.md for more on how version tagging works.
 
 3.1.38 (in development)
 -----------------------
+- The JS `err()` function will now bind to `console.error` by default rather
+  than `console.warning`.  For debugging/tracing/logging we recommend the
+  `dbg()` function instead. (#19326)
```

Comment:
Why do we recommend `dbg`?

Reply:
If you want to get tracing information it seems better than spewing errors to devtools which are lot more visually noisy.   I also like to be able to tell the difference between actuall errors and logging information.

Reply:
For anything being XXX_DEBUG settings we now use `dbg()` everywhere.   I guess I could have called it `trace()` or ` log()`?

Reply:
Wait, where does `dbg()`go, if not to devtools? `stderr` on the page's HTML?

Reply:
`dbg()` does go to devtools, but in the long run I would like to not to show up as either a warning or error (at least not by default), since we generate so many `dbg()` statements when we enabled debugging.

i.e. I would ilke `dbg()` to map to `console.log()` by default, or at least have some way to configure it so you don't get thousands of errors in the devtools console when you enable tracing.